puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/i18n-string-extraction-and-update-pipeline.yaml

```diff
@@ -0,0 +1,10 @@
+- job:
+    name: i18n-string-extraction-and-update-pipeline
+    project-type: pipeline
+    sandbox: true
+    triggers:
+      - timed: '30 4 */7 * *'
```

Comment:
I don't think you need to specify it twice (both here and groovy). Not sure which one would take precedence or would be preferred.